websites/A/AnimeYY/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.15",
+  "apiVersion": 1,
+  "author": {
+    "id": "432215088686956565",
+    "name": "xDoritos"
+  },
+  "contributors": [
+    {
+      "name": "Sladkaroww",
+      "id": "123123123"
+    }
+  ],
+  "service": "AnimeYY",
+  "description": {
+    "en": "AnimeYY is an online platform where you can stream and browse a wide variety of anime series and movies for free. It offers updated episodes, multiple genres, and an easy-to-use interface for discovering new shows."
+  },
+  "url": [
+    "animeyy.com",
+    "www.animeyy.com"
+  ],
+  "version": "1.0.0",
```

Comment:
Please add a `regExp` field. This will replace the functionality of the `url` field in the future.